tracer/build/_build/Build.Steps.cs

```diff
@@ -1312,6 +1316,24 @@ void CopyWaf(string architecture, IEnumerable<string> frameworks, AbsolutePath a
             }
         });
 
+    Target UpdateSnapshots => _ => _
+        .Description("Updates verified snapshots files with received ones")
+        .Executes(() =>
+        {
+            Logger.Info($"Snaps");
+
+            var snapshotsDirectory = Path.Combine(TestsDirectory, "snapshots");
+            var directory = new DirectoryInfo(snapshotsDirectory);
+            var files = directory.GetFiles("*.received*");
+
+            foreach (var file in files)
+            {
+                var source = file.FullName;
+                var dest = file.FullName.Replace("received", "verified");
```

Comment:
This will break if the phrase "received" exists in the snapshot name. I think we need to use something like:
```suggestion
            var files = directory.GetFiles("*.received.*");
            var suffixLength = "received".Length;

            foreach (var file in files)
            {
                var source = file.FullName;
                var fileName = Path.GetFileNameWithoutExtension(source);
                if (!fileName.EndsWith("received"))
                {
                    Logger.Warn($"Skipping file '{source}' as filename did not end with 'received'");
                }

                var trimmedName = fileName.Substring(0, fileName.Length - suffixLength);
                var dest = Path.Combine(directory.Name, $"{trimmedName}.{Path.GetExtension(source)}");
```

Reply:
Thanks